src/photos/ducks/backup/components/BackupHeader.jsx

```diff
@@ -3,14 +3,14 @@ import React from 'react'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
-import PhotoSync from 'photos/assets/illustrations/cozy-photo-sync.svg'
```

Comment:
`cozy-photo-sync.svg` is still used? Or can we remove it?

Reply:
Still used for permissions modal